app.ts

```diff
@@ -0,0 +1,187 @@
+
+	let rowsThatFit = amountOfRowsThatFit();
+
+	from = parseInt(toValue) + 1;
+	to = from + rowsThatFit;
+
+	if (to > 100000) {
+		window.alert("Nee, stout >.<,your search has exceeded the maximum rows")
+		from = 100000 - rowsThatFit;
+
+		to = 100000;
+	}
+	fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value = from.toString();
+	toValue = (<HTMLInputElement>document.getElementById("toValue")).value = to.toString();
+
+
+	callHeaders(from, to);
+}
+/**
+ *Executes when the user presses the previous button
+ */
+function pressedPrevious() {
+
+	let toValue = (<HTMLInputElement>document.getElementById("toValue")).value;
+	let fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value;
+	let rowsThatFit = amountOfRowsThatFit();
+
+	to = from - 1;
+	from = parseInt(fromValue) - rowsThatFit;
+	if (from <= -1) {
+		window.alert("Nee, stout >.<,your search has exceeded the minimum rows")
+		from = 0;
+		to = from + rowsThatFit;
+	}
+	fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value = from.toString();
+	toValue = (<HTMLInputElement>document.getElementById("toValue")).value = to.toString();
+	callHeaders(from, to);
+
+}
+
+
+/**
+ *Retrieves column names
+*/
+function callHeaders(from: number, to: number) {
+	fetch('http://localhost:2050/columns')
+		.then((resp) => {
+			return resp.json();
+		})
+		.then((columnsRequest) => {
+			let headerData = <string[]>columnsRequest;
+			create(headerData);
+			retrieveRows(from, to);
+		});
+}
+
+/**
+ *Retrieves the table rows with data
+ */
+function retrieveRows(from: number, to: number) {
+	let indexNumber = 0;
+
+	fetch('http://localhost:2050/records?from=' + from + '&to=' + to)
+		.then((resp) => {
+			return resp.json();
+		})
+		.then((rowsRequest) => {
+
+			//check if the status is 200(means everything is okay)
+			let columnData = <string[]>rowsRequest;
+
+
+			for (let j = 1; j < 15; j++) {
+				let newRow: HTMLElement = document.createElement('tr');
+				newRow.setAttribute('id', 'rowNumber' + j);
+				document.getElementById("intialTable")!.appendChild(newRow);
+				console.log("new row");
+
+				for (let i = 0; i < 11; i++) {
+					console.log("here");
+					let columnName = document.createElement('td');
+					console.log("evolving");
+					columnName.textContent = columnData[indexNumber];
+					console.log(columnData[indexNumber]);
+					document.getElementById("rowNumber" + j)!.appendChild(columnName);
+
+					indexNumber++;
+				}
+			}
+		});
+
+
+}
+
+
+function create(input: string[]) {
+	//create table
+	let tableRetrieved = <HTMLInputElement>document.getElementById("intialTable");
+	if (tableRetrieved != null) {
+		tableRetrieved.remove();
+	}
+	let table = document.createElement('table');
+	table.setAttribute('id', 'intialTable');
+	document.getElementById("body")!.appendChild(table);
+
+	let newRow = document.createElement('tr');
+	newRow.setAttribute('id', 'rowNumber0');
```

Comment:
could you set the ids in all of your HTML elements to be kebab case (eg. 'row-number-1'), not because this adds much value to THIS onboarding, but because it will add value to the future code that you do write.